Contributors.md

```diff
@@ -0,0 +1,21 @@
+# Maintainers
+| Name | GH ID | Expertise | Affiliation | Location |
+|---|---|---|---|---|
+| Huamin Chen | rootfs | Kepler Core | Red Hat | USA |
+| Marcelo Carneiro do Amaral | marceloamaral | Kepler Core | IBM | Japan |
+| Sunyanan Choochotkaew (Pang) | sunya-ch | Kepler Core | IBM | Japn |
+| Ji Chen | jichenjc | Testing | IBM | China |
+| Sam Yang | SamYuan1990 | CI | IBM | China |
+| William Caban Babilonia | williamcaban | Kepler Deployment | Red Hat | USA |
+| Kayi Liu | KaiyiLiu1234 | Kepler Model Server | Red Hat | Cananda |
+| Parul Singh | husky-parul  | Kepler Deployment and Documentation | Red Hat | USA |
+| Chen Wang | wangchen615 | Kepler Core | IBM | USA |
+
+# Reviewers
```

Comment:
@rootfs shouldn't we also be in the reviewers list?

Reply:
some clarity the roles is needed, hopefully this request can help https://github.com/sustainable-computing-io/kepler/issues/249

Reply:
@rootfs right and I think we also need to configure the owners

Kubernetes has a guideline for this [https://go.k8s.io/owners](https://go.k8s.io/owners)
```
approvers: a list of GitHub usernames or aliases that can /approve a PR
reviewers: a list of GitHub usernames or aliases that are good candidates to /lgtm a PR
```

Kubernetes has an example of this [here](https://github.com/kubernetes/kubernetes/blob/master/OWNERS)

Reply:
having that CI would be great :D